### ISSUE TITLE
accordionPanel: Ignores f:passThroughAttribute

### ISSUE THREAD

**Issue by vagelis-savvas** [primefaces/primefaces#8283]
**Describe the defect**
p:accordionPanel ignores f:passThroughAttribute, the specified attribute is not rendered in the final HTML document.

**Reproducer**
None.

**Environment:**
 - PF Version: _10.0_
 - JSF + version: _2.3_ (Mojarra 2.3.14.SP04)
 - Affected browsers: ALL

**To Reproduce**
Simply use a p:accordionPanel with an f:passThroughAttribute and observe that the named attribute and its value do not appear at the final HTML document.

**Expected behavior**
The passed through attribute and its value should appear in the rendered HTML document.

**Example XHTML**
```html
<p:accordionPanel id="accPanelTest">
    <f:passThroughAttribute name="data-my-attr" value="some value" />
    <p:tab title="Header I" />
</p:accordionPanel>
```

**Comment by melloware** [comment]
Thanks for reporting.  PR Submitted.

**Comment by vagelis-savvas** [comment]
Hello,
is this going to be resolved for PF 10.0.0 as well?
BR,
Vagelis

**Comment by melloware** [comment]
@vagelis-savvas probably only if you have a PRO account and request it from Primetek to be added to 10.0.0
